benchmark/citation/appnp.py

```diff
@@ -46,5 +49,14 @@ def forward(self, data):
 
 dataset = get_planetoid_dataset(args.dataset, args.normalize_features)
 permute_masks = random_planetoid_splits if args.random_splits else None
+print("appnp-{}-{}:".format(args.dataset, args.random_splits), end=' ')
 run(dataset, Net(dataset), args.runs, args.epochs, args.lr, args.weight_decay,
-    args.early_stopping, permute_masks)
+    args.early_stopping, args.inference, args.profile, permute_masks)
+
+if args.profile:
```

Comment:
Can we make this a function in the benchmark to avoid duplicated code? Alternatively, run this inside `run()`?

Reply:
Yes. The new function `rename_profile_file` is added into` torch_geometric.profile` with `trace_handler`. In `run()` of `train_eval.py`, the conv layer name cannot be obtained directly, need to change the conv layer to the same name in `Net`.